src/sidebar/panel.html

```diff
@@ -42,13 +43,20 @@
 
     <div id="footer-buttons">
       <button id="enable-sync"></button>
-      <a target="_blank" id="give-feedback"></a>
+      <div class="wrapper">
+        <button id="context-menu-button" class="mdl-js-button"></button>
+        <ul class="mdl-menu mdl-menu--top-right mdl-js-menu context-menu" data-mdl-for="context-menu-button">
+          <li id="give-feedback" class="mdl-menu__item context-menu-item"></li>
+          <li id="disable-sync" class="mdl-menu__item context-menu-item"></li>
```

Comment:
Let's remove this button and just have 1 "Feedback" button. the "sync" button will be added as part of a different PR.